packages/hash/api/src/graphql/typeDefs/knowledge/entity.typedef.ts

```diff
@@ -1,116 +1,41 @@
 import { gql } from "apollo-server-express";
 
-export const persistedEntityTypedef = gql`
-  interface PersistedEntity {
-    # These fields are repeated everywhere they're used because
-    # (a) GQL requires it - https://github.com/graphql/graphql-spec/issues/533
-    # (b) string interpolation breaks the code generator's introspection
-    #
-    # Could maybe use a custom schema loader to parse it ourselves:
-    # https://www.graphql-code-generator.com/docs/getting-started/schema-field#custom-schema-loader
-    #
-    # For now, _COPY ANY CHANGES_ from here to any type that 'implements Entity'
-    """
-    The id of the entity
-    """
-    entityId: ID!
-    """
-    The specific version of the entity
-    """
-    entityVersion: String!
-    """
-    The id of the account that owns this entity.
-    """
-    ownedById: ID!
-    """
-    Alias of ownedById - the id of the account that owns this entity.
-    """
-    accountId: ID!
-      @deprecated(reason: "accountId is deprecated. Use ownedById instead.")
-    """
-    The versioned URI of this entity's type.
-    """
-    entityTypeId: String!
-    """
-    The JSON object containing the entity's properties.
-    """
-    properties: JSONObject!
-  }
+export const entityWithMetadataTypedef = gql`
+  scalar EntityId
+  scalar EntityEditionId
+  scalar PropertyObject
+  scalar EntityMetadata
 
-  type UnknownPersistedEntity implements PersistedEntity {
-    # ENTITY INTERFACE FIELDS BEGIN #
-    """
-    The id of the entity
+  type EntityWithMetadata {
```

Comment:
Do we intend on just using the `Entity` scalar down the line? If so can we leave a `@todo` here? If not, why not?

Reply:
I think we should, yeah. I'll leave a todo. The reason why we aren't doing it yet is to allow for custom resolvers for pages/comments/etc. to allow migrating in pieces before switching to subgraphs

Reply:
Now that I think about it, we don't need this type to be here as it's not an interface type anymore

Reply:
I've addressed the todo and removed the explicit type in 029b8ed033c5361bc4e7b630bdb33b6db83c995a